df_py/util/dftool_module.py

```diff
@@ -507,6 +511,9 @@ def do_calc():
         save_challenge_rewards_csv(challenge_rewards, csv_dir)
 
     if arguments.SUBSTREAM == "predictoor":
+        if arguments.CHAINID is None:
```

Comment:
I think you can define that with argparse somehow, directly. And Instead of a ValueError it should behave exactly the same as the sys.exit provided by argparse